front/src/applications/operationalStudies/components/SimulationResults/SpeedSpaceChart/SpeedSpaceChart.jsx

```diff
@@ -189,6 +195,31 @@ export default function SpeedSpaceChart(props) {
     };
   }, [chart, trainSimulation, localSettings, resetChart, rotate]);
 
+  // to be removed after creating power restriction module
+  useEffect(() => {
+    setRestrictionPower('');
+    trainSimulation.electrificationConditions.forEach((elem) => {
+      if (elem.used_restriction)
+        setRestrictionPower(
+          `${t('speedSpaceSettings.powerRestriction')}: ${elem.used_restriction}`
+        );
+    });
+    if (!restrictionPower) {
+      trainSimulation.electrificationConditions.forEach((elem) => {
+        setRestrictionPower(
+          `${t('speedSpaceSettings.powerRestriction')} ${elem.seen_restriction} ${t(
+            'powerRestriction.waited'
+          )}, ${t('powerRestriction.incompatible')}`
+        );
+      });
+      if (!restrictionPower) {
+        setRestrictionPower(
+          `${t('speedSpaceSettings.powerRestriction')} ${t('powerRestriction.unknown')}`
```

Comment:
```suggestion
          ${t('powerRestriction.unknown')}`
```
Only use 1 traduction (bad practice, it will be broken if the speedSpaceSettings.powerRestriction is modified !). You should then modify powerRestriction.unknown: "Restriction de puissance inconnue"

Reply:
done